cmake/FindBROKER.cmake

```diff
@@ -18,13 +18,14 @@
 find_library(
   BROKER_LIBRARY
   NAMES broker
-  HINTS ${BROKER_ROOT_DIR}/lib ${BROKER_ROOT_DIR})
+  HINTS ${BROKER_ROOT_DIR} ${VAST_PREFIX}
```

Comment:
Not sure if I like that the FindBROKER module knows about variables that start with VAST.

Reply:
I know it's not the cleanest way to implement this, but putting it here reduces the complexity of the call site considerably.

Reply:
I'd rather have the complexity with a comment. Moving this here makes maintenance in the future harder.

Aside, do we _really_ need a FindBROKER module? Is the one shipping with broker with a supplied HINT at the call side not sufficient?

Reply:
`findPackage` does not support `HINT` in legacy mode.

Reply:
As discussed via direct messaging, we decided to remove `VAST_PREFIX` from the Broker search path.